tools/bin/hawqregister

```diff
@@ -460,12 +460,13 @@ if __name__ == '__main__':
                 do_not_move, files_update, sizes_update = True, files, sizes
                 files, sizes = [], []
         else:
-            for k, f in enumerate(files):
+            files_old, sizes_old = [f for f in files], [sz for sz in sizes]
```

Comment:
Notice here, alternative modification like `files_old, sizes_old = files, sizes` is wrong since python default copy is not a deep copy, it is just a pointer copy.